content/hi/security-chaos-engineering.md

```diff
@@ -0,0 +1,42 @@
+---
+title: सुरक्षा अव्यवस्था इंजीनियरिंग  (Security Chaos Engineering)
+status: Completed
+category: अवधारणा
+tags: ["security", "methodology", ""]
+---
+
+## यह क्या है
+
+
+सुरक्षा अव्यवस्था इंजीनियरिंग या एससीई एक ऐसी शाखा है जो की [सीमावर्ती प्रणाली](/chaos-engineering/) पर आधारित है।
+एससीई वितरित प्रणाली पर सक्रिय सुरक्षा प्रयोग करता है जिससे प्रणाली की क्षमता के विषय में विश्वास बढ़ता है ताकि यह
```

Comment:
Instead of 'जिससे प्रणाली की क्षमता के विषय में विश्वास बढ़ता है' we can write 'जिससे प्रणाली की क्षमता में विश्वास बढ़ता है'

Reply:
Changes made as per the suggestions. 
Thanks @jayesh-srivastava for reviewing up 💙.